src/coordination/replication_instance_client.cpp

```diff
@@ -117,6 +117,24 @@ auto ReplicationInstanceClient::DemoteToReplica() const -> bool {
   return false;
 }
 
+auto ReplicationInstanceClient::RegisterReplica(utils::UUID const &uuid,
+                                                struct ReplicationClientInfo replication_client_info) const -> bool {
```

Comment:
why everywhere struct before ReplicationClientInfo?

Reply:
ReplicationClientInfo was getter so I needed struct in front of type. I changed ReplicationClientInfo func to be called GetReplicationClientInfo to avoid name collision